### PR TITLE
adjust Storage. remove App class, Readme file

### PR DESCRIPTION
**Pull request by fyarkiy**

### CODE REVIEW

**Review by MichaelShevchenko — APPROVED** [review]
Great job!

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
 "# OwnProject_simpleShop"
```
Comment (dmytro-hryhoruk) [inline README.md]:
```suggestion
# OwnProject_simpleShop
```

**Review by dmytro-hryhoruk — COMMENTED** [review]
Let's use these tags for readme https://github.com/sandino/Markdown-Cheatsheet

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
-"# OwnProject_simpleShop" 
+# OwnProject_simpleShop
+*Project purpose*
```
Comment (RomanDubovskyi) [inline README.md]:
```suggestion
## Project purpose
```

File: README.md [hunk 3]
```diff
@@ -1 +1,45 @@
...
+
+Implementation of basic internet shop functionality. It has 
+- user authentication  
+- user authorisation
+- product set up (add and delete)
+- add and remove products to shopping cart
+- convert shopping cart into order
+
+*Project structure*
```
Comment (RomanDubovskyi) [inline README.md]: same here

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**